components/brave_shields/browser/tracking_protection_service_browsertest.cc

```diff
@@ -158,3 +168,63 @@ IN_PROC_BROWSER_TEST_F(TrackingProtectionServiceTest, TrackerReferencedFromUntru
 
   EXPECT_EQ(browser()->profile()->GetPrefs()->GetUint64(kTrackersBlocked), 1ULL);
 }
+
+#if BUILDFLAG(BRAVE_STP_ENABLED)
+IN_PROC_BROWSER_TEST_F(TrackingProtectionServiceTest, StorageTrackingBlocked) {
+  if (!TrackingProtectionHelper::IsSmartTrackingProtectionEnabled()) {
```

Comment:
It is possible to just add another test fixture and override `SetUpDefaultCommandLine()` to enable the feature and make sure that the test always runs